LGTM

there's a test which times out in https://jenkins.dockerproject.org/job/Docker-PRs/32176/console though
```
 --- FAIL: Test (971.18s)
08:08:24 panic: DockerSuite.TestHealth test timed out after 5m0s [recovered]
08:08:24 	panic: DockerSuite.TestHealth test timed out after 5m0s
```

--- a/daemon/daemon.go
+++ b/daemon/daemon.go
@@ -1055,7 +1055,7 @@ func (daemon *Daemon) reloadClusterDiscovery(config *Config) error {
 	}
 	netOptions, err := daemon.networkOptions(daemon.configStore, nil)
 	if err != nil {
-		logrus.Warnf("Failed to reload configuration with network controller: %v", err)
+		logrus.WithError(err).Warnf("failed to get options with network controller")
 		return nil
 	}
 	err = daemon.netController.ReloadConfiguration(netOptions...)

